ui/src/funding/contract.ts

```diff
@@ -27,31 +32,119 @@ const addresses = {
 };
 
 // Get the address of the Pool Contract for the given environment
-export function poolAddress(environment: ethereum.Environment): string {
+export function daiTokenAddress(environment: ethereum.Environment): string {
   switch (environment) {
     case ethereum.Environment.Local:
-      return addresses.pool.local;
+      return addresses.dai.local;
     case ethereum.Environment.Ropsten:
-      return addresses.pool.ropsten;
+      return addresses.dai.ropsten;
```

Comment:
These changes are just coded being moved around to have it grouped by the type of contract.